people/JanaRod.json

```diff
@@ -0,0 +1,12 @@
+{
+  "key": "janarod",
+  "name": "Ing. Jana Röderová Ph.D.",
+  "nick": "Jana",
+  "email": "jana.roderova@mild.blue",
+  "title_cs": "Datová analytička",
+  "title_en": "Data Analyst",
+  "title_de": "Daten Analyst",
+  "short_desc_en": "Jana received her doctorate in Microbiology at UCT and worked at the Institute of Microbiology of the CAS. She uses her experience of working with scientific texts in the preparation of clinical studies and is involved in data analysis.",
+  "short_desc_de": "Jana erhielt ihren Ph.D. in Mikrobiologie an der UCT und arbeitete am Mikrobiologisches Institut des Tschechische Akademie der Wissenschaften. Sie nutzt ihre Erfahrung in der Arbeit mit wissenschaftlichen Texten in der Vorbereitung klinischer Studien und ist in die Datenanalyse involviert.",
+  "short_desc_cs": "Jana získala doktorský titul v oboru Mikrobiologie na VŠCHT a pracovala na Mikrobiologickém ústavu AV ČR. Svých zkušeností práce s odbornými texty využívá při přípravě klinických studií a podílí se na datové analýze."
```

Comment:
`z` prace s odbornymi texty?

Reply:
nebo jako v AJ, zkusenosti z vedeckeho vyzkumu.

A treba pridat uplne na zaver
...datove analyze a obecne vedecke cinnosti Mild Blue

Jestli ti to prijde dobre. Pak pripadne upravit i ostatni jazyky